text/src/autogluon/text/automm/data/process_text.py

```diff
@@ -397,4 +484,15 @@ def __call__(
         per_sample_text = {
             per_column_name: per_column_text[idx] for per_column_name, per_column_text in all_text.items()
         }
-        return self.build_one_token_sequence_from_text(per_sample_text)
+        return self.build_one_token_sequence_from_text(per_sample_text, is_training)
+
+    #nlpaug couldn't be serialize, so not saving augmenter for now
+    def __deepcopy__(self, memo):
+        cls = self.__class__
+        result = cls.__new__(cls)
+        memo[id(self)] = result
+        
+        for k, v in self.__dict__.items():
+            if(k!="train_augmenter"):
+                setattr(result, k, deepcopy(v, memo))
```

Comment:
What will be `result.train_augmenter` if we do this. Should we reconstruct it manually via
```
result.train_augmenter = construct_augmenter(result.train_augment_types)
```

Also, we need to test it.

Reply:
deepcopy(naw.SynonymAug()) gives "cannot pickle '_io.BufferedReader' object". So as a work around, I am not saving the train_augmenter, and construct again during load.

Reply:
I can understand the error. Basically, you can reconstruct `result.train_augmenter` after you have deep-copied the other attributes. Also, for support pickle/unpickle, you can also override `__getstate__` and `__setstate__`.